docs/docs/recipes.md

```diff
@@ -853,6 +853,129 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing data from Contentful
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start/)
+- A [Contentful account](https://www.contentful.com/)
+- The [Contentful CLI](https://www.npmjs.com/package/contentful-cli) installed
+
+#### Directions
+
+1. Log in to Contentful with the CLI and follow the steps. It will help you create an account if you don't have one.
+
+```shell
+contentful login
+```
+
+2. Create a new space. Make sure to save the space ID given to you at the end of the command.
+
+```shell
+contentful space create --name 'Gatsby example'
+```
+
+3. Seed the new space with example blog content using the new space ID returned from the previous command, in place of `<space ID>`.
+
+```shell
+contentful space seed -s '<space ID>' -t blog
+```
+
+For example, with a space ID in place: `contentful space seed -s '22fzx88spbp7' -t blog`
+
+4. Create a new access token for your space.
+
+```shell
+contentful space accesstoken create -s '<space ID>' --name 'Example token'
+```
+
+5. Install the `gatsby-source-contentful` plugin in your Gatsby site:
+
+```shell
+npm install --save gatsby-source-contentful
+```
+
+6. Edit the file `gatsby-config.js` to enable the `gatsby-source-contentful` plugin. You should strongly consider using [environment variables](/docs/environment-variables/) to store your space ID and token for security purposes.
+
+```javascript:title=gatsby-config.js
+plugins: [
+  {
```

Comment:
I don't think this comment is as effective as it could be, as it appears related to the following code instead of outside of it. I'd recommend something like this:

```suggestion
   // add to array along with any other installed plugins
   // highlight-start
   {


```

Reply:
yeh thats better